### PR TITLE
Hacktoberfest 2020: Add style improvements for Project Euler Problem 03

### PR DESCRIPTION
**Pull request by archaengel**
### **Describe your change:**

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

Related: #2786

### CODE REVIEW

File: project_euler/problem_03/sol2.py [hunk 1]
```diff
@@ -41,15 +41,19 @@ def solution(n):
         raise TypeError("Parameter n must be int or passive of cast to int.")
     if n <= 0:
         raise ValueError("Parameter n must be greater or equal to one.")
+
     prime = 1
     i = 2
+
```
Comment (dhruvmanila) [inline project_euler/problem_03/sol2.py:47]: No need of unnecessary newlines. Please remove them wherever they were added.

File: project_euler/problem_03/sol3.py [hunk 2]
```diff
@@ -41,23 +41,26 @@ def solution(n: int) -> int:
         raise TypeError("Parameter n must be int or passive of cast to int.")
     if n <= 0:
         raise ValueError("Parameter n must be greater or equal to one.")
+
     i = 2
     ans = 0
+
```
Comment (dhruvmanila) [inline project_euler/problem_03/sol3.py:47]: No need of unnecessary newlines. Please remove them wherever they were added.

**Review by dhruvmanila — CHANGES_REQUESTED** [review]

**Comment by archaengel** [comment]
I've removed the unnecessary newlines. Let me know if there's anything else I should address!

**Review by dhruvmanila — APPROVED** [review]
Thank you for your contribution.

**Comment by archaengel** [comment]
Thanks for the review!

**Pull request merged**